x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/index.tsx

```diff
@@ -94,18 +93,21 @@ const StepRuleActionsComponent: FC<StepRuleActionsProps> = ({
     [application]
   );
 
-  const onSubmit = useCallback(
-    async (enabled: boolean) => {
-      if (setStepData) {
-        setStepData(RuleStep.ruleActions, null, false);
-        const { isValid: newIsValid, data } = await submit();
-        if (newIsValid) {
-          setStepData(RuleStep.ruleActions, { ...data, enabled }, newIsValid);
-          setMyStepData({ ...data, isNew: false } as ActionsStepRule);
-        }
+  useEffect(() => {
```

Comment:
This `useEffect` looks strange. The field should get its value from `defaultValue` defined on the form.

Reply:
Good call, this was moved to `defaultValue`.